### PR TITLE
added blog post about dependency with wildcards

### PR DESCRIPTION
**Pull request by WarrenFaith**
@RaghavSood @AdamMc331 @Rashiq @TristanWiley @codeMagic23 @MobileOak 

any feedback appreciated!

### CODE REVIEW

**Comment by WarrenFaith** [comment]
Why ever the CI failed... anyone a clue? I haven't changed anything beside adding a new post

File: _posts/2016-01-27-Library-versioning-done-right.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+With the wildcard, the time of the build might become a critical aspect. You start your build at `1am` and test the app. Everything works. You build your release at `2am` and publish it. A day later you get huge amounts of bad rating and crash reports in your developer console.
+
+Now you start to evaluate how this could happen only to find out (probably after digging for a long time) that at `1:30am` the developer of one of your dependencies published a new version where he/she, mistakenly, introduced some bad bugs. This resulted in a updated build at `2am` where you weren't aware of this changes.
+
+## Invisible Changes
+
+The worst wildcard would be if there is no version defined at all but only the `+`. You would never realize that there are improvements or changes in an automatically updated version. You might have workarounds implemented for existing bugs that are already fixed since ages. The API you use might be already deprecated. There might be a cooler way to get stuff done. You will probably never find out until you specifically check your code for deprecate warnings or the API of the library changes in a way that your build breaks.
+
+## You are in not in charge!
```
Comment (fuho) [inline _posts/2016-01-27-Library-versioning-done-right.md]: Two ins.

Comment (fuho) [inline _posts/2016-01-27-Library-versioning-done-right.md]: Two ins.

Comment (fuho) [inline _posts/2016-01-27-Library-versioning-done-right.md]: Two ins.

Comment (fuho) [inline _posts/2016-01-27-Library-versioning-done-right.md]: Two ins.

**Pull request merged**